### ISSUE TITLE
Provincial-Level Setting of Income Elasticity Coefficients for the Steel Sector

### ISSUE THREAD

**Issue by moshang27** [umd-cgs/gcam-china#4]
Dear GCAM-China team:
      Can the steel sector, like the cement sector, set income elasticity coefficients at the provincial level without modifying the underlying code? Thanks!

**Comment by ouyang363** [comment]
yes, basically just follow iron_steel_incelas_ssp2_china.xml structure in GCAM-China-v7 or later
